src/ipc/encoder.cpp

```diff
@@ -11,9 +11,9 @@ namespace ipc {
     size_t total_size = HEADER_SIZE + payload.size();
     std::vector<uint8_t> data(total_size);
 
-    auto* header = reinterpret_cast<ipc::header*>(data.data());
-    std::copy(ipc::MAGIC.begin(), ipc::MAGIC.end(), header->s.magic);
-    header->s.version = ipc::VERSION;
+    auto* header = reinterpret_cast<polybar::ipc::header*>(data.data());
```

Comment:
I think renaming the variable `msg_header` and just using `header*` as the type makes more sense. Explicitly having the polybar namespace there will likely not survive any future refactoring.

Reply:
I have renamed it to `msg_header` and updated the commit.